CONTRIBUTING.md

```diff
@@ -284,13 +284,13 @@ Various hints for developers
 - We are using codecov which has extensions for the popular browsers
   (Firefox, Chrome) which annotates pull requests on github regarding changed coverage.
 
-### Useful Flags
+### Useful Environment Variables
+Refer datalad/config.py for information on how to add these environment variables to the config file and their naming convention
```

Comment:
On Tue, 14 Jun 2016, Debanjum wrote:
>  > +Refer datalad/config.py for information on how to add these environment variables to the config file and their naming convention
>    Should I remove these lines or remove them for now ?
just keep them -- we will make it really (it is already somewhat but not
"hardened") factual soon ;)

-- 
Yaroslav O. Halchenko
Center for Open Neuroscience     http://centerforopenneuroscience.org
Dartmouth College, 419 Moore Hall, Hinman Box 6207, Hanover, NH 03755
Phone: +1 (603) 646-9834                       Fax: +1 (603) 646-1419
WWW:   http://www.linkedin.com/in/yarik